### DISCUSSION TITLE
status around the issue "ERROR Duplicate macro name "

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD-flow-scripts#1130] [Q&A] [answered]
I am getting the `Duplicate macro name` error reported in #942 and #944  when I run `make clean_all && make finish` These two issues are now closed but I do not understand what the fix to the issue is.

**Comment by oharboe** [comment]
Please file a github issue with exact reproduction details.

**Comment by gkamendje** [comment, marked as answer]
Duplicate entries in Klayout.lyt was the root cause. See #1135 for details
